pom.xml

```diff
@@ -2082,7 +2095,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-report-plugin</artifactId>
-        <version>2.22.2</version>
+        <version>2.18</version>
```

Comment:
latest version is 3.2.2... any reason we can't go to the latest here?  (2.18 is from 2014..  nearly 10 years old at this point.)

Reply:
after spending several hours on it and going through a variety of rabbit-holes "No", not if we want jacoco 0.8.11 (latest) to work